src/build.js

```diff
@@ -43,6 +45,74 @@ export async function generateBuildMap(configObj) {
   return await iterateDir(configObj.inputDir);
 }
 
+export async function generateBundleOutputObj(buildMap) {
+  const filenames = buildMap.map(buildPair => buildPair[0].pathname);
```

Comment:
nit: May be clearer to destructure here. E.g.:

```js
buildMap.map(([goodNameForFirstInPair]) => goodNameForFirstInPair.pathname);
```

Reply:
Definitely, cleared this up